roles/ansible/tower/config-ansible-tower/tasks/install.yml

```diff
@@ -35,11 +35,53 @@
     args:
       chdir: "{{ ansible_tower_dir }}"
 
-  - name: "Download and Install the 'oc' client for OpenShift interactions"
-    shell: curl {{ ansible_tower_oc_download_url }} | tar -C /bin/ -xzf -
-    args:
-      warn: False
+  - name: "Download 'oc' client for OpenShift interactions"
```

Comment:
The `unarchive` module supports remote files (i.e.: URLs), Is there a particular reason why we don't use it? Since `remote_src` is set below, it appears that perhaps the intent was to use it? If so, I'd suggest combining the two tasks, as such:

```
    - unarchive:
        src: "{{ ansible_tower_oc_download_url }}"
        dest: /bin
        remote_src: yes
        mode: 0755
```

Reply:
... same for the helm part too of course